### DISCUSSION TITLE
Question about IDAES boiler flowsheet recon example

### DISCUSSION THREAD

**Discussion by Yunshan-Liu67** [IDAES/idaes-pse#1273] [Q&A]
Hi all, 
I'm currently looking at [the boiler flowsheet recon example](https://idaes.github.io/examples-pse/latest/Examples/Advanced/DataRecon/boiler_flowsheet_recon_doc.html)  and find there are warnings coming out of the scaling of the model reporting there are some scaling_factor contain component keys that are not exported as part of the NL file. I'm wondering what's causing this issue and why we could ignore these warnings.
Thank you very much for your help!

Best Regards,
Yunshan

**Comment by andrewlee94** [comment]
@Yunshan-Liu67 Those warning can be ignored. The reason those are appearing is because IDAES is "borrowing" Pyomo scaling suffixes to store data that the suffix was not designed for. Originally, Pyomo ignored these silently but v6 started to emit warning for unexpected suffixes. We are working on Pyomo to fix this, but it will be a few months at least before this is done.
